fluidreview/api.py

```diff
@@ -235,14 +235,19 @@ def parse_webhook_user(webhook):
         user = profile.user
     if webhook.award_id is not None:
         personal_price = webhook.award_cost if webhook.amount_to_pay is None else webhook.amount_to_pay
-        klass = Klass.objects.get(klass_key=webhook.award_id)
-        if personal_price is not None:
-            user.klass_prices.update_or_create(
-                klass=klass,
-                defaults={'price': personal_price}
-            )
+        klass = Klass.objects.filter(klass_key=webhook.award_id).first()
+        if klass:
+            if personal_price is not None:
```

Comment:
This won't get run if the klass & bootcamp need to be created, can you reorder the code to something like this:
```python
    if webhook.award_id is not None:
        klass = Klass.objects.filter(klass_key=webhook.award_id).first()
        if not klass:
            klass_info = FluidReviewAPI().get('/awards/{}'.format(webhook.award_id)).json()
            bootcamp = Bootcamp.objects.create(title=klass_info['name'])
            klass = Klass.objects.create(bootcamp=bootcamp, title=klass_info['description'], klass_key=klass_info['id'])
        personal_price = webhook.award_cost if webhook.amount_to_pay is None else webhook.amount_to_pay
        if personal_price is not None:
            user.klass_prices.update_or_create(
                klass=klass,
                defaults={'price': personal_price}
            )
        else:
            user.klass_prices.filter(klass=klass).delete()
```